README.md

```diff
@@ -311,7 +311,11 @@ are mandatory to configure the Broker client:
 - `BROKER_TOKEN` - The Snyk Broker token, obtained from your Container registry integration settings (app.snyk.io).
 - `BROKER_CLIENT_URL` - The URL of your broker client (including scheme and - port) used by container registry agent to call back to Snyk.
 - `CR_AGENT_URL` - The URL of your container registry agent (including scheme and - port) to which brokered requests would be forwarded.
-- `CR_CREDENTIALS` - Base64-encoded credentials json used by agent to access container registry.
+- `CR_TYPE` - The container registry type as listed in supporter registries, for example "DockerHub", "GoogleCR", "ArtifactoryCR".
+- `CR_BASE` - The hostname of the container registry api to connect to, for example: "cr.host.com".
```

Comment:
Maybe use upper case 'API'?

Reply:
I'm aiming to be consistent with the documentation on the Snyk Docs website. I'd prefer for them to be exactly the same in this pull request.